db/logs_test.go

```diff
@@ -31,3 +32,32 @@ func TestLogsDB(t *testing.T) {
 	req.Equal("INFO", logs[0].LogLevel)
 	req.Equal("Sub", logs[0].Subsystem)
 }
+
+func TestLogsDBCleanup(t *testing.T) {
+	req := require.New(t)
+	ctx := context.Background()
+
+	sqldb := CreateTestTmpDB(t)
+	require.NoError(t, CreateAllBoostTables(ctx, sqldb, sqldb))
+
+	ldb := NewLogsDB(sqldb)
+
+	deals, err := GenerateDeals()
+	req.NoError(err)
+	deal := deals[0]
+
+	err = ldb.InsertLog(ctx, &DealLog{DealUUID: deal.DealUuid, CreatedAt: time.Now(), LogLevel: "INFO", LogParams: "params", LogMsg: "Test", Subsystem: "Sub"})
+	req.NoError(err)
+
+	logs, err := ldb.Logs(ctx, deal.DealUuid)
+	req.NoError(err)
+	req.Len(logs, 1)
+
+	// Provide a negative duration to delete everything from today
+	err = ldb.CleanupLogs(ctx, -1)
```

Comment:
I'd suggest that you
- insert two logs, one with created at set to more than one day ago, and with created at set to the current time
- call `ldb.CleanupLogs(ctx, 1)`
- verify that only the older log was removed